docs/index.md

```diff
@@ -19,8 +19,7 @@ enthusiasts to enable everyone to build better maps.
 
 ## Quick start
 
-The easiest way to get started is to have a look at easy to use the **[HERE Studio](https://studio.here.com/)** web application to build and publish maps. Give it a try, it's completely free and open to use for everyone - you only need a HERE Account. Have a look at our
-**[documentation](studio/topics/index.md)** for find out more about it.
+The easiest way to get started is to have a look at easy to use the **[HERE Studio](https://studio.here.com/)** web application to build and publish maps. Give it a try, it's completely free and open to use for everyone - you only need a HERE Account.
```

Comment:
When we're about to fix errors 'have a look at easy to use the ... web application' should perhaps be 'have a look at the easy to use ... web application'.